daemon/mgr/cri_streaming.go

```diff
@@ -0,0 +1,50 @@
+package mgr
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	k8snet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubernetes/pkg/kubelet/server/streaming"
+)
+
+func newStreamServer(ctrMgr ContainerMgr, address, port string) (streaming.Server, error) {
+	if address == "" {
+		// If address is "", we would use the host's default interface.
+		addr, err := k8snet.ChooseBindAddress(nil)
```

Comment:
can we just implement a ChooseBindAddress method? not import a whole apimachinery pkg?